packages/just-task/src/package/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './findDependents';
+export * from './findGitRoot';
+export * from './findPackageRoot';
```

Comment:
These two are available from `workspace-tools`, so it's probably better to just import them from there instead. (`just-task` should also be updated to use the `workspace-tools` versions ideally.)

Reply:
Thanks for the feedback!